reporter/s3.go

```diff
@@ -103,34 +99,41 @@ func (w S3Writer) Write(rs ...models.ScanResult) (err error) {
 	return nil
 }
 
+// ErrBucketCheckExist : bucket cannot check the existence because s3:ListBucket or s3:ListAllMyBuckets is not allowed
+var ErrBucketCheckExist = xerrors.New("bucket cannot check the existence because s3:ListBucket or s3:ListAllMyBuckets is not allowed")
```

Comment:
The sentence "bucket cannot check the existence" sounds a little strange because the subject "bucket" do "check" its existence. For example, "bucket existence cannot be checked" may be slightly better??

Reply:
adopted.
https://github.com/future-architect/vuls/compare/8f49be19277966e328387507b7f30a889b83bc34..b9fa4280920ecbb76b1d133014f738f878664b8e